app/(authorized)/lines/(drawer)/_layout.tsx

```
@@ -1,21 +1,26 @@
   DrawerItemList,
 } from "@react-navigation/drawer";
 import { useMutation } from "@tanstack/react-query";
-import { Redirect, useRouter } from "expo-router";
+import { Redirect, Stack, useRouter } from "expo-router";
 import { Drawer } from "expo-router/drawer";
+import { Box } from "native-base";
 import { FC } from "react";
-import { ActivityIndicator } from "react-native";
+import { ActivityIndicator, Pressable } from "react-native";
 
 type LayoutPropsType = {};
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)